Dockerfile

```
@@ -0,0 +1,11 @@
+FROM python:3.10.3-alpine3.15
```

Is the version intentionally so specific?

---

```suggestion
FROM python:alpine
```

Resolves to [roughly the same version](https://hub.docker.com/_/python) and will get updated to new stable Python releases and Alpine release:

```
3.10.4-alpine3.15, 3.10-alpine3.15, 3-alpine3.15, alpine3.15, 3.10.4-alpine, 3.10-alpine, 3-alpine, alpine
```

---

Well it needed to be 3.10 and there was a long list to choose from, all of them having weird names and non of them just being 3.10-slim. So, if you can find a more generic small version that works I'd be happy to change it. I don't know the differences between all these strange named versions so I just picked something that had a cool name.

---

Great, then I will use that!  Thanks.